src/vl-map-select-action.js

```diff
@@ -63,11 +63,12 @@ export class VlMapSelectAction extends VlMapAction {
   }
 
   _createAction(layer) {
-    return new VlSelectAction(layer, (args) => {
-      this._onSelect(args);
-    }, {
+    const callback = (args) => this._onSelect(args);
+    const options = {
       style: this._style,
       cluster: (this._cluster != undefined),
-    });
+    };
+    const select = new VlSelectAction(layer, callback, options);
+    return select;
```

Comment:
Kunt het ineens returnen ipv in een const te steken

Reply:
Inderdaad, blijkbaar niet gereverted na debugging.